### ISSUE TITLE
Add flag for saving logits

### ISSUE THREAD

**Issue by VeritasJoker** [hassonlab/247-pickling#140]
Add a flag in the Makefile for saving logits when generating embeddings

**Comment by hvgazula** [comment]
excellent suggestion...let me think of it..or do you want to create a PR 😛

**Comment by hvgazula** [comment]
decided to save logits always as it is a by-product of running model inference anyway.
